modules/test/playwright/tests/osb-faro-web/utils/navigation.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {Page} from '@playwright/test';
+
+import {faroConfig} from '../faro.config';
+
+export async function navigateTo(page: Page, pageName: string) {
+	await page.getByRole('link', {name: pageName}).first().click();
+}
+
+export async function navigateToACPage(page: Page) {
```

Comment:
Can we rename function name from navigateToACPage to navigateToWorkspace? And we can have a new parameter to pass the name of workspace and have one by default:

```export async function navigateToWorkspace(page: Page, workspaceName: string = 'FARO-DEV-liferay Liferay Demo Enterprise Plan') {```

Reply:
Ohhh nice, thanks @interaminense  🎉